upi/azure/azuredeploy.json

```diff
@@ -0,0 +1,1072 @@
+    "infraPublicIpAddressId" : "[resourceId('Microsoft.Network/publicIPAddresses', variables('infraPublicIpName'))]",
+    "infraLbId" : "[resourceId('Microsoft.Network/loadBalancers', variables('infraLoadBalancerName'))]",
+    "infraLbFrontEndConfigId" : "[concat(variables('infraLbId'), '/frontendIPConfigurations/AppLoadBalancerFrontEnd')]",
+    "infraLbBackendPoolId" : "[concat(variables('infraLbId'),'/backendAddressPools/AppLoadBalancerBackend')]",
+    "infraLbHttpProbeId" : "[concat(variables('infraLbId'),'/probes/httpProbe')]",
+    "infraLbHttpsProbeId" : "[concat(variables('infraLbId'),'/probes/httpsProbe')]",
+    "infraLbCockpitProbeId" : "[concat(variables('infraLbId'),'/probes/cockpitProbe')]",
+    "StorageAccountPersistentVolume" : "[concat('sapv', resourceGroup().name)]",
+    "registryStorageName" : "[concat('sareg', resourceGroup().name)]",
+    "apiVersion" : "2015-06-15",
+    "apiVersionCompute" : "2015-06-15",
+    "apiVersionNetwork" : "2016-03-30",
+    "tmApiVersion" : "2015-11-01",
+    "apiVersionStorage" : "2015-06-15",
+    "apiVersionLinkTemplate" : "2015-01-01",
+    "sshKeyPath" : "[concat('/home/','core','/.ssh/authorized_keys')]",
+    "intLoadBalancerName" : "[concat(resourceGroup().name, 'intlb')]",
+    "intLbId" : "[resourceId('Microsoft.Network/loadBalancers', variables('intLoadBalancerName'))]",
+    "intLbBackendPoolId" : "[concat(variables('intLbId'),'/backendAddressPools/masters')]",
+    "intLbProbeId" : "[concat(variables('intLbId'),'/probes/lbprobe')]",
+    "intLbProbeId2" : "[concat(variables('intLbId'),'/probes/probe22623')]",
+    "intLbFrontEndConfigId" : "[concat(variables('intLbId'), '/frontendIPConfigurations/loadBalancerFrontEnd')]"
+  },
+  "resources" : [
+    {
+      "type" : "Microsoft.Storage/storageAccounts",
+      "name" : "[variables('nodeStorageName')]",
+      "apiVersion" : "[variables('apiVersion')]",
+      "location" : "[variables('location')]",
+      "properties" : {
+        "accountType" : "[variables('vmSizesMap')[parameters('nodeVmSize')].storageAccountType]"
+      }
+    },
+    {
+      "apiVersion" : "[variables('apiVersion')]",
+      "type" : "Microsoft.Network/virtualNetworks",
+      "name" : "[variables('virtualNetworkName')]",
+      "location" : "[variables('location')]",
+      "tags" : {
+        "displayName" : "VirtualNetwork"
+      },
+      "properties" : {
+        "addressSpace" : {
+          "addressPrefixes" : [
+            "[variables('addressPrefix')]"
+          ]
+        },
+        "subnets" : [
+          {
+            "name" : "[variables('nodesubnetName')]",
+            "properties" : {
+              "addressPrefix" : "[variables('nodesubnetPrefix')]"
+            }
+          },
+          {
+            "name" : "[variables('mastersubnetName')]",
+            "properties" : {
+              "addressPrefix" : "[variables('mastersubnetPrefix')]"
+            }
+          }
+        ]
+      }
+    },
```

Comment:
if the user needs to use their own vnets/subnets.. what's the usual flow for Azure users.